### DISCUSSION TITLE
GNN explainer for Graph classification

### DISCUSSION THREAD

**Discussion by lsvannur** [pyg-team/pytorch_geometric#2874] [Q&A]
Does Torch geometric support for explanations to Graph classification tasks?

Any help in explaining graphs or point to any link reference.

**Comment by wsad1** [comment]
Check out the `explain_graph` function in the [GNNExaplainer](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.models.GNNExplainer) class. It learns a node and edge mask that explains a prediction.
